src/app/pages/fund/components/fiat-provider-item.tsx

```diff
@@ -4,17 +4,20 @@ import { FastCheckoutBadge } from './fast-checkout-badge';
 import { ZeroPercentFeesBadge } from './zero-percent-fees-badge';
 import { FundAccountTile } from './fund-account-tile';
 
-const description = 'Available in the US, South Korea, Europe and +33 countries';
+const availableInsideUSDescription = 'Available in the US and other countries';
```

Comment:
Same here, [Microsoft has guidelines for this](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN), as does @zone117x also have opinions lol

Strictly speaking, as a two better acronym, this passes the rules, but imo it's still better to avoid two letter acronyms where possible. `USD` is a common abbreviation, so literally read this as _available inside usd..._.

Suggest: `availableInsideUsaDescription`

Reply:
I see that now, thx. Will change.